### PR TITLE
feat : db added.

### PR DESCRIPTION
**Pull request by Onksami**
- mongodb has been added.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,6 +1,24 @@
-import { app } from './app.js';
-import { PORT } from './utils/config.js';
+import { app } from "./app.js";
+import { PORT } from "./utils/config.js";
+
+app.post("/assignments", async (req, res) => {
```
Comment (DagnaSchmidt) [inline server/index.js]: please delete both functions, or move them to controllers/assignments.js - index.js should only run server

File: db_test.js
Comment (DagnaSchmidt) [inline db_test.js:1]: please create tests folder in server and move this file there

File: server/index.js (hunk 1 above)
Comment (Onksami) [reply server/index.js]: functions removed.

File: db_test.js
Comment (Onksami) [reply db_test.js:1]: test folder created and db_test.js has been moved there

**Review by DagnaSchmidt — APPROVED** [review]

**Pull request merged**